hardhat.config.ts

```diff
@@ -18,32 +18,34 @@
  *
  */
 
-import * as dotenv from "dotenv";
-import { HardhatUserConfig } from "hardhat/config";
-import "@nomicfoundation/hardhat-toolbox";
-import "hardhat-contract-sizer";
-import "@openzeppelin/hardhat-upgrades";
+import { HardhatUserConfig } from 'hardhat/config';
+import '@openzeppelin/hardhat-upgrades';
+import '@nomiclabs/hardhat-ethers';
+import dotenv from 'dotenv';
 
 dotenv.config();
 
 const config: HardhatUserConfig = {
   solidity: {
-    version: "0.8.17",
+    version: '0.8.6',
     settings: {
       optimizer: {
         enabled: true,
-        /// Note: A “runs” parameter of “1” will produce short but expensive code.
-        ///       In contrast, a larger “runs” parameter will produce longer but more gas efficient code.
-        ///       The maximum value of the parameter is 2**32-1. So it could change as required.
         runs: 13000,
       },
     },
   },
   networks: {
-    hardhat: {
-      allowUnlimitedContractSize: true,
-    },
-  },
+    hederaTestnet: {
```

Comment:
Can you add a config for local node in here and consider defaulting to that setup instead

Reply:
All good now. Local node is installed and working in this starter! I also added some tests too to demonstrate testing the smart contracts